package.json

```diff
@@ -17,6 +17,7 @@
     "test": "jest --notify --watch",
     "test:coverage": "jest --coverage",
     "test:coverage:changes": "jest --coverage --changedSince=origin/main",
+    "test:coverage:cloud-data-sources": "jest --coverage -- ./public/app/plugins/datasource/cloudwatch/** ./public/app/plugins/datasource/cloud-monitoring/** ./public/app/plugins/datasource/grafana-azure-monitor-datasource/**",
```

Comment:
I won't use this. I will instead grep the lines from the full coverage report
cc. @andresmgot

Reply:
Looks like the full coverage report will have each subdirectory reported in a separate entry. Extracting the number and calculating coverage for each plugin's sub-directory might be a little bit hack-y.